custom_components/keymaster/system_health.py

```diff
@@ -0,0 +1,25 @@
+"""Provide info to system health."""
+from homeassistant.components import system_health
+from homeassistant.core import HomeAssistant, callback
+
+from .const import DOMAIN
+
+
+@callback
+def async_register(
+    hass: HomeAssistant, register: system_health.SystemHealthRegistration
+) -> None:
+    """Register system health callbacks."""
+    register.async_register_info(system_health_info)
+
+
+async def system_health_info(hass):
+    """Get info for the info page."""
+    client = hass.data[DOMAIN]
+
+    return {
+        "zwave_integration": client["zwave_integration"],
+        "network_status": hass.states.get(
+            f"binary_sensor.{client['network_sensor']}"
+        ).state,
```

Comment:
```suggestion
        "network_status": hass.states[
            f"binary_sensor.{client['network_sensor']}"
        ].state,
```
If you are worried the state won't be there, this may cause an error either way. Also wondering if we want to translate the state to Connected/Disconnected instead of On/Off?

Reply:
Could do that in a follow up PR.